src/main/drivers/flash/flash_m25p16.c

```diff
@@ -106,6 +106,9 @@ struct {
     // Datasheet: https://www.winbond.com/resource-files/w25q64jv%20spi%20%20%20revc%2006032016%20kms.pdf
     { 0xEF4017, 133, 50, 128, 256 }, // W25Q64JV-IQ/JQ
     { 0xEF7017, 133, 50, 128, 256 }, // W25Q64JV-IM/JM*
+    // Datasheet: https://www.winbond.com/resource-files/W25Q64JW%20RevE%2003102021%20Plus.pdf
+    { 0xEF6017, 133, 50, 128, 256 }, // W25Q64JW-IQ/JQ
```

Comment:
This should be 104, not 133.
![image](https://github.com/user-attachments/assets/f38af59d-7656-41a5-832e-160f63313f48)

Reply:
yes, it should be 104MHz